app/src/main/java/ch/epfl/sweng/eventmanager/repository/data/Spot.java

```diff
@@ -0,0 +1,47 @@
+package ch.epfl.sweng.eventmanager.repository.data;
+
+import com.google.android.gms.maps.model.LatLng;
+import com.google.maps.android.clustering.ClusterItem;
+
+/**
+ * Represents a place into the event
+ *
+ * @author Robin Zbinden (274236)
+ * @author Stanislas Jouven (260580)
+ */
+public class Spot implements ClusterItem {
+
+    /**
+     * The name for the spot
+     */
+    private String title;
+    /**
+     * The description of the spot
+     */
+    private SpotType spotType;
+    /**
+     * The position of the spot
+     */
+    private LatLng position;
```

Comment:
LatLong typo ?

Reply:
No, LatLng is a type in the Google Map API

Reply:
Ok sry I read too quickly didn't saw it was a type :)